docs/conf.py

```diff
@@ -160,7 +160,7 @@
 
 # List of patterns, relative to source directory, that match files and
 # directories to ignore when looking for source files.
-exclude_patterns = ['_build']
+exclude_patterns = ['_build', 'howto/operator/gcp/_partials']
```

Comment:
Didn't you try to sneak in this "partials" change with some other commit? ;>

Reply:
Yes. I do it. This file is already on apache/master

Reply:
I updated a PR.